utilities/utilities.css

```diff
@@ -1,168 +1,170 @@
 /* Textbox */
 
 .textbox {
-
-    /* border: solid 1px black; */
-    /* padding-left: 24px;
+  /* border: solid 1px black; */
+  /* padding-left: 24px;
     padding-right: 24px; */
+}
+
+.title {
+  font-size: 21px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 32px;
+  letter-spacing: 0px;
+  text-align: left;
+  margin-bottom: 0px;
+  margin-top: 10px;
+}
+.subtitle {
+  margin-bottom: 10px;
+  margin-top: 0px;
+  color: var(--font-light);
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 20px;
+  letter-spacing: 0px;
+  text-align: left;
+}
+.text {
+  color: var(--font-light);
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0px;
+  text-align: left;
+  margin: 14px 0px;
+}
+.CTA-text {
+  margin-top: 0px;
+  margin-bottom: 14px;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 36px;
+  letter-spacing: 0px;
+  text-align: left;
+}
+
+/* Buttons */
+
+.button-primary {
+  cursor: pointer;
+  font-size: 1.5rem;
+  background-color: var(--primary);
+  padding: 10px 23px;
+  border: none;
+  border-radius: 12px;
+  color: white;
+  height: fit-content;
+  border: solid 2px var(--primary);
+  font-weight: 500;
+}
+.button-primary:hover {
+  border: solid 2px var(--primary);
+  background-color: transparent;
+  color: var(--primary);
+}
+.button-secondary {
+  cursor: pointer;
+  font-size: 1.5rem;
+  background-color: var(--grey);
+  padding: 10px 23px;
+  border: none;
+  border-radius: 12px;
+  color: var(--primary);
+  height: fit-content;
+  border: solid 2px var(--primary);
+  font-weight: 500;
+}
+.button-secondary:hover {
+  border: solid 2px var(--primary);
+  background-color: var(--primary);
+  color: white;
+}
+
+/* button vertical */
+
+.button-vertical :nth-child(n) {
+  width: 100%;
+}
+.button-vertical :nth-child(n + 2) {
+  margin-top: 5px;
+}
+
+/* button horizontal */
+
+.button-horizontal {
+}
+
+/* Background colors */
+
+.has-primary {
+  background-color: var(--primary);
+}
+.has-secondary {
+  background-color: var(--secondary);
+}
+.has-blue {
+  background-color: var(--blue);
+}
+.has-red {
+  background-color: var(--red);
+}
+.has-green {
+  background-color: var(--green);
+}
+.has-yellow {
+  background-color: var(--yellow);
+}
+.has-black {
+  background-color: black;
+}
+.has-white {
+  background-color: white;
+}
+
+/* colors */
+
+.is-blue {
```

Comment:
use has/is overall. Don't use both has/is

Reply:
I am using "has" for background color and "is" for colors property, That's why I am using both.